runtime/src/lib.rs

```diff
@@ -1260,60 +1172,54 @@ construct_runtime! {
         NodeBlock = polkadex_primitives::Block,
         UncheckedExtrinsic = UncheckedExtrinsic
     {
-        System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
-        Utility: pallet_utility::{Pallet, Call, Event} = 1,
-        Babe: pallet_babe::{Pallet, Call, Storage, Config, ValidateUnsigned} = 2,
-        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
-        Authorship: pallet_authorship::{Pallet, Call, Storage, Inherent} = 4,
-        Indices: pallet_indices::{Pallet, Call, Storage, Config<T>, Event<T>} = 5,
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 6,
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 7,
-        ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Pallet, Call, Storage, Event<T>, ValidateUnsigned} = 8,
-        Staking: pallet_staking::{Pallet, Call, Config<T>, Storage, Event<T>} = 9,
-        Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 10,
-        Democracy: pallet_democracy::{Pallet, Call, Storage, Config, Event<T>} = 11,
-        Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 12,
-        TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 13,
-        Elections: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>, Config<T>} = 14,
-        TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 15,
-        Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event, ValidateUnsigned} = 16,
-        Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 17,
```

Comment:
Please don't remove the numbering of pallets in Runtime. It's required for Ethereum contracts for PDEX migration to work

Reply:
Re enumerate or leave holes?

Reply:
Re enumerate. It will break otherwise.